src/commands/distribute/groups/show.ts

```diff
@@ -0,0 +1,131 @@
+      Promise.all([distributionGroupMembersRequestResponse, basicReleasesDetailsRequestResponse].map((p) => p.catch(() => Promise.resolve()))));
+
+    let distributionGroupMembers: models.DistributionGroupUserGetResponse[];
+    try {
+      debug(`Getting users of distribution group ${this.distributionGroup}`);
+      let response = await distributionGroupMembersRequestResponse;
+      if (response.response.statusCode < 400) {        
+        distributionGroupMembers = response.result;
+      } else {
+        throw response.response.statusCode;
+      }
+    } catch (error) {
+      if (error === 404) {
+        return failure(ErrorCodes.InvalidParameter, `distribution group ${this.distributionGroup} was not found`);
+      } else {
+        debug(`Failed to get list of distribution group members - ${inspect(error)}`);
+        return failure(ErrorCodes.Exception, "failed to retrieve list of distribution group users");
+      }
+    }
+
+    let basicReleasesDetails: models.BasicReleaseDetails[];
+    try {
+      debug(`Getting releases details for distribution group ${this.distributionGroup}`);
+      let response = await basicReleasesDetailsRequestResponse;
+      if (response.response.statusCode < 400) {
+        basicReleasesDetails = response.result;
+      } else {
+        throw response.response.statusCode;
+      }
+    } catch (error) {
+      debug(`Failed to get releases for the distribution group - ${inspect(error)}`);
+      return failure(ErrorCodes.Exception, "failed to retrieve releases details for the distribution group");
+    }
+
+    // creating requests for extended release information
+    const limit = pLimit(10);
+    const releasesDetailsRequestsResponses = <Array<Promise<ClientResponse<models.ReleaseDetails>>>> basicReleasesDetails.map((basicReleaseDetails) => {
+      return limit(() => clientRequest<models.ReleaseDetails>((cb) => client.distribute.getRelease(basicReleaseDetails.id.toString(), app.ownerName, app.appName, cb)));
+    });
+
+    // show spinner and wait for releases details requests to finish
+    await out.progress("Loading releases details...", 
+      Promise.all(releasesDetailsRequestsResponses.map((p) => p.catch(() => Promise.resolve()))));
+
+    const releasesDetails: models.ReleaseDetails[] = [];
+    for (let i = 0; i < releasesDetailsRequestsResponses.length; i++) {
+      const releasesDetailsRequestResponse = releasesDetailsRequestsResponses[i];
+      try {
+        const response = await releasesDetailsRequestResponse;
+        if (response.response.statusCode < 400) {
+          debug(`Succeeded to get release details for release ${basicReleasesDetails[i].id}`);
+          releasesDetails.push(response.result);
+        } else {
+          throw response.response.statusCode;
+        }
+      } catch (error) {
+        debug(`Failed to get release details for release ${basicReleasesDetails[i].id} - ${inspect(error)}`);        
+      }
+    }
+
+    // TODO implement expected output
```

Comment:
It needs to be re-done once the expected output will be specified.
Right now it outputs all the fields of DistributionGroupUserGetResponse and ReleaseDetails. It is also not JSON friendly for now. 
@JoshuaWeber , @izikl , can you, please, provide the example of the expected output for this command?